lib/internal/url.js

```diff
@@ -1365,6 +1372,10 @@ function getPathFromURLPosix(url) {
   return decodeURIComponent(pathname);
 }
 
+/**
+ * @param {string | URL} path 
```

Comment:
I believe these need to be wrapped in parentheses:

```suggest
* @param {(string | URL)} path
```

[jsdoc→types→type union](https://jsdoc.app/tags-types.html)

Reply:
Core is targeting JSDoc support in TypeScript not jsdoc the documentation tool. We could add them, but they are not actually needed. Just be sure to stick with TS support and not break the practical usage of the tags is my thoughts. So don't use jsdoc.org but instead defer to TS